package.json

```diff
@@ -3,7 +3,7 @@
   "name": "lockwise",
   "id": "lockbox@mozilla.com",
   "version": "2.2.4-alpha",
```

Comment:
I'm not sure if manifest changes will be applied for existing users unless you bump the version number.

Reply:
it won't.  I wanted to get this mechanics done first, then do an actual release (including the latest from pontoon).